packages/amplify-graphql-transformer-core/src/types/import-appsync-api-types.ts

```diff
@@ -29,3 +29,5 @@ export type RDSConnectionSecrets = TransformerSecrets & {
 
 export const MYSQL_DB_TYPE = 'MySQL';
 export const DDB_DB_TYPE = 'DDB';
+
+export type ModelDatasourceType = 'MySQL' | 'DDB';
```

Comment:
nit, assuming these should always stay in sync: 
```suggestion
export type ModelDatasourceType = MYSQL_DB_TYPE | DDB_DB_TYPE;
```

Reply:
I tried this but this typescript wasn't happy because MYSQL_DB_TYPE and DDB_DB_TYPE are const values and not types.